shared/wallets/routes.js

```diff
@@ -17,7 +16,7 @@ const routeTree = makeRouteDefNode({
     },
     linkExisting: {
       children: {},
-      component: MaybePopupHoc(isMobile)(LinkExisting),
+      component: LinkExisting,
```

Comment:
Moved the `MaybePopup` into `LinkExisting` like how it was done with `ExportSecretKey` and `ReceiveModal`. This means the popup looks in storybook how it would look in the app!